client/src/components/AppComponent.jsx

```diff
@@ -133,28 +134,28 @@ export default class AppComponent extends React.Component {
                 </div>
 
                 <Switch>
-                    <Route
+                    <ProtectedRoute
```

Comment:
I protected all links in nav bar from unauth.user=>> policy for protection is in ProtectedRoute class.